python/cuml/cuml/tests/test_sgd.py

```diff
@@ -22,9 +22,15 @@
 from cuml.solvers import SGD as cumlSGD
 
 
+def test_sgd_penalty_none_deprecated():
+    with pytest.warns(FutureWarning):
```

Comment:
```suggestion
    with pytest.warns(FutureWarning, match="penalty='none' is deprecated"):
```
because I'm paranoid about matching the wrong warning :D